src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -46,61 +46,90 @@ export type ImportAccountsProps = {
 }
 
 type TableRowProps = {
-  accountId: AccountId
+  accountIds: AccountId[]
   accountNumber: number
   asset: Asset
-  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
+  onAccountIdsActiveChange: (accountIds: AccountId[], isActive: boolean) => void
+}
+
+type TableRowAccountProps = {
+  accountId: AccountId
+  asset: Asset
 }
 
 const disabledProps = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
 
+const TableRowAccount = forwardRef<TableRowAccountProps, 'div'>(({ asset, accountId }, ref) => {
+  const translate = useTranslate()
+  const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
```

Comment:
As spotted by @0xApotheosis in https://github.com/shapeshift/web/pull/6798/files#r1590517374, not sure about this guy (though not related to this PR). This is supposed to reflect the subtitle for the account i.e:

https://github.com/shapeshift/web/blob/a8d28d3e4bbb4dc9cac17351fe9ac54d72faa807/src/pages/Accounts/components/AccountEntryRow.tsx#L55

Reply:
Nice. I've linked this comment in the ticket
https://github.com/shapeshift/web/issues/6837